src/components/tiles/Tile.tsx

```diff
@@ -1,75 +1,127 @@
-import React from 'react';
+import React, { FC } from 'react';
 import { Dimensions, TouchableOpacity, View } from 'react-native';
 import { Text } from 'react-native-paper';
 import { useStyles } from '../../hooks/useStyles';
-import { createStyles, Theme } from '../BrandConfigProvider';
+import { createStyles } from '../BrandConfigProvider';
+import TileSelectIcon from './icons/tile-select-chevron.svg';
+import { SvgProps } from 'react-native-svg';
+import LinearGradient from 'react-native-linear-gradient';
 import { tID } from '../../common/testID';
 
-export interface TileProps {
-  onPress?: () => void;
-  id: string;
+interface TileProps {
   title: string;
-  mode?: 'halfLength' | 'fullLength';
+  Icon?: React.FC<SvgProps>;
+  onPress?: () => void;
+  id?: string;
+  testID?: string;
   children?: React.ReactNode;
-  styles?: TileStyles;
 }
 
-export function Tile({
-  onPress,
-  id,
-  title,
-  mode = 'fullLength',
-  children,
-  styles: instanceStyles,
-}: TileProps) {
-  const { styles } = useStyles(defaultStyles, instanceStyles);
+const ViewTile: FC<TileProps> = ({ Icon, title, id, testID }) => {
```

Comment:
Why switch to `FC`?  We can leave this as a function and still utilize `TileProps`.